### PR TITLE
Align aegis to hopsworks 0.9

### PR DESCRIPTION
**Pull request by cbotsikas**
I've included missing files

### CODE REVIEW

File: aegis-web/yo/app/index.html [hunk 1]
```diff
@@ -178,7 +181,6 @@
     <script src="scripts/js/hamburger.js"></script>
     <script src="scripts/js/ace/ace.js"></script>
     <script src="scripts/js/ace/mode-json.js"></script>
-    <script src="scripts/js/graph-settings.js"></script>
```
Comment (cbotsikas) [inline aegis-web/yo/app/index.html:181]: This looks to be a forgotten reference. The file doesn't exist.

File: aegis-web/yo/package.json [hunk 2]
```diff
@@ -1,17 +1,15 @@
 {
   "name": "aegis",
   "version": "0.0.1",
-  "dependencies": {
-    "apexcharts": "^3.3.0"
-  },
+  "dependencies": {},
```
Comment (cbotsikas) [inline aegis-web/yo/package.json:4]: this is not needed here, it's included with bower

File: aegis-web/yo/package.json [hunk 3]
```diff
@@ -1,17 +1,15 @@
...
   "repository": {},
   "devDependencies": {
-    "bower-npm-resolver": "^0.9.1",
     "globule": "^1.0.0",
     "grunt": "^0.4.2",
     "grunt-autoprefixer": "^2.0.0",
     "grunt-concurrent": "^2.3.0",
     "grunt-connect-proxy": "^0.2.0",
+    "serve-static": "^1.13.2",
```
Comment (cbotsikas) [inline aegis-web/yo/package.json:12]: I've rearranged the dependencies to match the hopsworks.

**Review by cbotsikas — COMMENTED** [review]
There are also some other differences but i didn't include those. 
Here's the list of files:
- [/pom.xml](https://github.com/aegisbigdata/hopsworks/blob/dev/hopsworks-web/pom.xml#L71-L90)
- [/src/main/webapp/WEB-INF/web.xml](https://github.com/aegisbigdata/hopsworks/blob/dev/hopsworks-web/src/main/webapp/WEB-INF/web.xml#L64-L96)
- [AngularFileManagerServlet.java](https://github.com/aegisbigdata/hopsworks/blob/dev/hopsworks-web/src/main/java/io/hops/hopsworks/api/airflow/AngularFileManagerServlet.java)

**Comment by o-alex** [comment]
the fixes above are good

File: aegis-web/yo/app/index.html (hunk 1 above)
Comment (o-alex) [reply aegis-web/yo/app/index.html:181]: yes, it seems it is not used anymore

**Review by o-alex — APPROVED** [review]
the changes for the dependencies are fine.
the html/css additions should also work fine, if you want to support those parts too. We already added the js part, when we also updated the js for the REST endpoint changes.

**Pull request merged**